pkg/rbac/roles.go

```diff
@@ -0,0 +1,75 @@
+package rbac
+
+import (
+	"fmt"
+	"github.com/tektoncd/triggers/pkg/apis/triggers/v1alpha1"
+	rbacv1 "k8s.io/api/rbac/v1"
+	"k8s.io/apimachinery/pkg/api/errors"
+	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+	"k8s.io/client-go/kubernetes"
+)
+
+const (
+	// RolePostfix is the postfix added to the name of the role that will be created to review trigger resources
+	RolePostfix string = "-trigger-role"
+	// RoleBindingPostfix is the postfix added to the name of the role binding that will be bound to the EventListener service account
+	RoleBindingPostfix string = "-trigger-rolebinding"
+)
+
+// AddRoleBinding creates a role and role binding in the EventListener namespace for the EventListener's ServiceAccount to view Triggers resources.
+// Both the role and role binding have owner references on the EventListener.
```

Comment:
nit: `have owner references on the EventListener` sounds backwards to me, since the EventListener owns the Role & RoleBinding. Rephrase ideas: `have the EventListener in their list of owner references` or just `are owned by the EventListener`?